src/components/AttachmentModal.js

```diff
@@ -351,7 +350,7 @@ function AttachmentModal(props) {
                         <AttachmentCarousel
                             report={props.report}
                             onNavigate={onNavigate}
-                            source={tryResolveUrlFromApiRoot(props.source)}
+                            source={props.source}
```

Comment:
Will this actually work once deployed? Will you see pics on staging sent from production? Pics from production sent from staging?